docs/users-guide/Defining-Components.adoc

```diff
@@ -2212,8 +2212,8 @@ the container (e.g., the size of the data payload), and binary data
 representing a list of serialized *records*.
 A record is a unit of data.
 For a complete specification of the container format, see the 
-https://fprime.jpl.nasa.gov/latest/documentation/reference/api/cpp/html/_fw_dp.html[documentation]
-on F Prime framework support for data products.
+https://fprime.jpl.nasa.gov/devel/documentation/reference/sdd/Fw/Dp/docs/sdd/[F 
```

Comment:
This will be able to change to https://fprime.jpl.nasa.gov/devel/Fw/Dp/docs/sdd/ once the website is updated

Reply:
Fixed! This change is great.